src/ontology/cl-edit.owl

```diff
@@ -2532,6 +2533,10 @@ AnnotationAssertion(rdfs:label obo:IAO_0000700 "preferred_root")
 
 AnnotationAssertion(rdfs:label obo:IAO_0100001 "term replaced by"^^xsd:string)
 
+# Annotation Property: <http://purl.obolibrary.org/obo/cl#added_for_HCA> (<http://purl.obolibrary.org/obo/cl#added_for_HCA>)
```

Comment:
not sure what this is? should this be removed?

Reply:
(this came in when I fixed the conflict, btw)

Reply:
It's fine.  Added by Paola in another PR & fixed by me.